sycl/test-e2e/ESIMD/unified_memory_api/Inputs/atomic_update_slm.hpp

```diff
@@ -681,6 +697,12 @@ int test_with_mask(queue q) {
         test_fp_types_and_sizes<ImplLSCFmin, UseMask, UsePVCFeatures, UseAcc>(
             q);
 
+   // TODO: fadd/fsub are emulated in the newer driver, but do not pass validation.
```

Comment:
Can you please create an internal tracker for this case?

Reply:
Done.